### PR TITLE
Fix/confirm dialog buttons

### PR DESCRIPTION
**Pull request by SamHellawell**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6600,9 +6600,9 @@ combined-stream@^1.0.6, combined-stream@^1.0.8, combined-stream@~1.0.6:
   dependencies:
     delayed-stream "~1.0.0"
 
-"comlinkjs@git://github.com/ronhe/comlink.git#dist":
+"comlinkjs@https://github.com/ronhe/comlink.git#dist":
```
Comment (mike-parkhill) [inline yarn.lock:6603]: are these updated intentional?

File: src/design-system/buttons.js [hunk 2]
```diff
@@ -18,16 +20,25 @@ export const runAfterInteractions =
...
 
 export const useAsyncCallback = func => {
+  const isActive = useRef(true);
   const [loading, setLoading] = useState();
   const callback = () => {
     setLoading(true);
     runAfterInteractions(() => {
       Promise.resolve(func()).finally(() => {
-        setLoading(false);
+        if (isActive.currentsActive) {
```
Comment (mike-parkhill) [inline src/design-system/buttons.js:29]: is this ever going to be true? How does it get set?

Comment (maycon-mello) [reply src/design-system/buttons.js:29]: The default state is true, once it's unmounted it will be set to false

Comment (maycon-mello) [reply src/design-system/buttons.js:29]: but i think it should be `isActive.current` instead

File: yarn.lock (hunk 1 above)
Comment (SamHellawell) [reply yarn.lock:6603]: yeah, github has disabled this: https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git

File: src/design-system/buttons.js (hunk 2 above)
Comment (SamHellawell) [reply src/design-system/buttons.js]: not sure how this typo happened... fixed

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**